src/fees.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @module
+ * A library of functions, each of which employs a different weighing of trade-offs
+ * in suggesting a transaction fee.
+ *
+ * All functions should return a promise that resolves to a number of stroops.
+ */
+
+/**
+ * Fetch the bare minimum fee.
+ * @returns {Promise<number>} Minimum fee in stroops.
+ */
+export function fetchBaseFee() {
+  return Promise.resolve(100);
```

Comment:
Is there a constant somewhere to use instead of 100?

Reply:
Since we're adding this in `js-stellar-sdk` can we get this value from the latest ledger (`/ledgers?order=desc&limit=1`):
```json
"base_fee_in_stroops": 100,
```
This value can change in a future so I think we shouldn't hardcode it. (It's hardcoded in `js-stellar-base`) because it doesn't have Horizon API.

Reply:
There is, but we can't import it until `stellar-base` bumps version!

Reply:
@bartekn Your point is a good one, and requires a shift from using `StellarSdk.Fees` to putting the functions on `StellarSdk.Server` instances. That change is up now.